parl/common/error_handling.py

```diff
@@ -0,0 +1,55 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class LastElementError(Exception):
+    """
+    Raised when the last element or an element with non-zero game status is
+    sampled.
+
+    Attributes:
+        message(string): error message
+    """
+
+    def __init__(self, idx, status):
+        self.message = 'The element at {}' .format(idx)
+        if status:
+            self.message += ' has game status: {}'.format(status)
+        else:
+            self.message += ' is the last element.'
+
+
+def check_error(error_type, *args):
```

Comment:
Do you think it's better to split the three cases into three functions like check_type_error(), check_last_exp_error(), and check_value_error() ? This could be more clear, at least there won't be args[0], args[1], args[2] in the code.

Reply:
I think you are right.